spring-boot-admin-server-ui/src/main/frontend/views/instances/threaddump/threads-list.vue

```diff
@@ -105,195 +139,230 @@
+        const { start, end } = this.getTimeExtent(timelines);
+        const width = this.$el
+          .querySelector(".threads__timeline")
+          .getBoundingClientRect().width;
+        const totalSeconds = Math.floor(width / maxPixelsPerSeconds);
+        const x = d3
+          .scaleTime()
+          .range([0, width])
+          .domain([start, Math.max(start + (totalSeconds + 1) * 1000, end)]);
 
-          d3.select('.threads__scale')
-            .attr('width', width)
-            .call(d3.axisBottom(x)
+        d3.select(".threads__scale")
+          .attr("width", width)
+          .call(
+            d3
+              .axisBottom(x)
               .ticks(Math.max(2, Math.floor(width / 50)))
-              .tickFormat(d => moment(d).format('HH:mm:ss'))
-            );
+              .tickFormat((d) => moment(d).format("HH:mm:ss"))
+          );
 
-          Object.entries(timelines).forEach(([threadId, value]) => {
-            const svg = d3.select(`#thread-${threadId}`).attr('width', width);
-            const d = svg.selectAll('rect').data(value.timeline);
+        Object.entries(timelines).forEach(([threadId, value]) => {
+          const svg = d3.select(`#thread-${threadId}`).attr("width", width);
+          const d = svg.selectAll("rect").data(value.timeline);
 
-            d.enter()
-              .append('rect')
-              .attr('id', 'rect-threadid-' + threadId + '-start-' + value.timeline[value.timeline.length - 1].start)
-              .attr('class', d => `thread thread--${d.threadState.toLowerCase()}`)
-              .merge(d)
-              .attr('height', '2em')
-              .attr('x', d => x(d.start))
-              .transition(150)
-              .attr('width', d => Math.max(x(d.end) - x(d.start), x(d.start + 500) - x(d.start)));
+          d.enter()
+            .append("rect")
+            .attr(
+              "id",
+              "rect-threadid-" +
+                threadId +
+                "-start-" +
+                value.timeline[value.timeline.length - 1].start
+            )
+            .attr(
+              "class",
+              (d) => `thread thread--${d.threadState.toLowerCase()}`
+            )
+            .merge(d)
+            .attr("height", "2em")
+            .attr("x", (d) => x(d.start))
+            .transition(150)
+            .attr("width", (d) =>
+              Math.max(x(d.end) - x(d.start), x(d.start + 500) - x(d.start))
+            );
 
-            d3.selectAll('#rect-threadid-' + threadId + '-start-' + value.timeline[value.timeline.length - 1].start)
-              .on('click', (event, d) => this.showThreadDetails({threadId: threadId, start: d.start}))
-         });
+          d3.selectAll(
+            "#rect-threadid-" +
+              threadId +
+              "-start-" +
+              value.timeline[value.timeline.length - 1].start
+          ).on("click", (event, d) =>
+            this.showThreadDetails({ threadId: threadId, start: d.start })
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/ulischulte/spring-boot-admin/pullRequest?prid=8294305)
```suggestion
            this.showThreadDetails({ threadId, start: d.start })
```